lib/index.js

```diff
@@ -447,33 +469,185 @@ MapboxGeocoder.prototype = {
    * Get the language to use in UI elements and when making search requests
    *
    * Look first at the explicitly set options otherwise use the browser's language settings
-   *
-   * @returns {String} The language used by the geocoder
+   * @param {String} language Specify the language to use for response text and query result weighting. Options are IETF language tags comprised of a mandatory ISO 639-1 language code and optionally one or more IETF subtags for country or script. More than one value can also be specified, separated by commas.
+   * @returns {MapboxGeocoder} this
    */
-  getLanguage: function(){
+  setLanguage: function(language){
     var browserLocale = navigator.language || navigator.userLanguage || navigator.browserLanguage;
-    return this.options.language || browserLocale;
+    this.options.language = language || this.options.language || browserLocale;
+    return this;
   },
 
   /**
-  * Get the text to use as the search bar placeholder
-   *
-   * If placeholder is provided in options, then use options.placeholder
-   * Otherwise, if language is provided in options, then use the localized string of the first language if available
-   * Otherwise use the default
-   *
-   * @returns {String} the value to use as the search bar placeholder
-   * @private
+   * Get the language to use in UI elements and when making search requests
+   * @returns {String} The language(s) used by the plugin, if any
    */
-  _getPlaceholderText: function(){
-    if (this.options.placeholder) return this.options.placeholder;
-    if (this.options.language){
-      var firstLanguage = this.options.language.split(",")[0];
-      var language = subtag.language(firstLanguage);
-      var localizedValue = localization.placeholder[language];
-      if (localizedValue)  return localizedValue;
-    }
-    return 'Search';
+  getLanguage: function(){
+    return this.options.language;
+  },
+
+  /**
+   * Get the zoom level the map will move to when there is no bounding box on the selected result
+   * @returns {Number} the map zoom
+   */
+  getZoom: function(){
+    return this.options.zoom;
+  },
+
+  /**
+   * Set the zoom level 
+   * @param {Number} zoom On geocoded result what zoom level should the map animate to when a `bbox` isn't found in the response. If a `bbox` is found the map will fit to the `bbox`.
```

Comment:
Small copy suggestion:

```suggestion
   * @param {Number} zoom The zoom level that the map should animate to when a `bbox` isn't found in the response. If a `bbox` is found the map will fit to the `bbox`.
```